### PR TITLE
linode settings adv configs

### LINKED ISSUE #215: Implement Linode detail settings Advanced Profile list

### PR DESCRIPTION
**Pull request by na3d**
Closes #215

### CODE REVIEW

**Comment by na3d** [comment]
screen shot 2016-08-19 at 3 46 22 pm

**Comment by SirCmpwn** [comment]
Delete should be lighter grey. It just looks like text.

Can you implement actually deleting disks here, too? That's part of the scope of this task imo.

**Comment by na3d** [comment]
screen shot 2016-08-23 at 9 23 21 am

File: src/linodes/settings/components/ConfigPanel.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            <div className="input-group pull-right">
+              <a
+                href={`/linodes/${linode.id}/configs/create`}
+                className="btn btn-default"
+              >
+                Add a config
+              </a>
+            </div>
+          </div>
+          {content}
```
Comment (SirCmpwn) [inline src/linodes/settings/components/ConfigPanel.js]: You need to wrap this in a col-sm-12 or such

**Comment by eatonphil** [comment]
Could you use the same table spacing that is used in the linode table view? Or is this using the same spacing? It has more vertical padding than the mockups anyway.

**Comment by na3d** [comment]
The side spacing is the same, the vertical wasn't.  Thank you.

**Comment by na3d** [comment]
screen shot 2016-08-23 at 10 19 53 am

**Comment by SirCmpwn** [comment]
Closer, but the padding is still off. Colors also don't look right?

**Comment by na3d** [comment]
I see what you mean now with the padding.

I made the table row darker than the original screen, but any darker then it makes the text hard to see. It is already darker than the Linode table view.
screen shot 2016-08-23 at 11 00 33 am

**Comment by eatonphil** [comment]
No need to darken it I think. Use the same background color as is used in the linode table view. The mockups are weird here.

**Comment by na3d** [comment]
I think it is fair to say that it looks a little weird having the table background and the background around it being the same color.

**Comment by na3d** [comment]
Also, we have such changes in background in other areas which give an indication of interactivity.  So I think it holds a minor but important role to have those differences as well.

**Comment by na3d** [comment]
Before I commit this up, yes or no?
screen shot 2016-08-23 at 2 03 17 pm

**Comment by SirCmpwn** [comment]
I like the second version.

**Comment by eatonphil** [comment]
Looks fine

**Pull request merged**